### PR TITLE
adoc content converted to DocBook XML

### PR DESCRIPTION
**Pull request by csymons-suse**
supersedes PR 772
additional information and PDF in
PR 775

### CODE REVIEW

File: xml/networking-pcipt_sriov_config.xml [hunk 1]
```diff
@@ -115,6 +115,115 @@
...
+      </para>
+      <para>
+       For example, it is common to have a <literal>nouveau</literal> driver
+       from when the node was installed. This driver is a graphics driver for
+       Nvidia-based GPUs. It must be blacklisted as shown in this example.
+      </para>
+<screen>&prompt.ardana;echo 'blacklist nouveau' &gt;&gt; /etc/modprobe.d/nouveau-default.conf</screen>
+      <para>
+       The file location and its contents are important; the filename is your
+       choice. Other drivers can be blacklisted in the same manner, possibly
```
Comment (asettle) [inline xml/networking-pcipt_sriov_config.xml]: the filename is your choice? Like, what you call the file? If so, maybe rephrase to "the name of the file is your choice." Just so it's clear :)

File: xml/networking-pcipt_sriov_config.xml [hunk 2]
```diff
@@ -115,6 +115,115 @@
...
+       including Nvidia drivers.
+      </para>
+     </step>
+     <step>
+      <para>
+       On the host, <literal>iommu_groups</literal> should be enabled. To check
```
Comment (asettle) [inline xml/networking-pcipt_sriov_config.xml]: Probably a question for the OP, but is this insinuating that it is already enabled and you should check that? Or is it saying that you need to enable it now?

File: xml/networking-pcipt_sriov_config.xml [hunk 3]
```diff
@@ -115,6 +115,115 @@
...
+       if IOMMU is enabled:
+      </para>
+<screen>&prompt.root;virt-host-validate
+.....
+QEMU: Checking if IOMMU is enabled by kernel
+: WARN (IOMMU appears to be disabled in kernel. Add intel_iommu=on to kernel cmdline arguments)
+.....</screen>
+      <para>
+       To modify the kernel cmdline as suggested in the warning, edit the file
```
Comment (asettle) [inline xml/networking-pcipt_sriov_config.xml]: Should we expand this to fully say either CLI or command line? Or do we normally say cmdline?

File: xml/networking-pcipt_sriov_config.xml (hunk 2 above)
Comment (asettle) [reply xml/networking-pcipt_sriov_config.xml]: @sjamgade ^^

Comment (sjamgade) [reply xml/networking-pcipt_sriov_config.xml]: this is a kernel feature and user might have it enabled already, since its usage is not limited to OpenStack. 
But because that is needed for pci passthrough, this is line is mentioning that it needs to be enabled. Followed by how to check it.

File: xml/networking-pcipt_sriov_config.xml (hunk 3 above)
Comment (csymons-suse) [reply xml/networking-pcipt_sriov_config.xml]: This usage is consistent with the immediately preceding : WARN message
: WARN (IOMMU appears to be disabled in kernel. Add intel_iommu=on to kernel cmdline arguments)

Should stay as it is

File: xml/networking-pcipt_sriov_config.xml (hunk 1 above)
Comment (csymons-suse) [reply xml/networking-pcipt_sriov_config.xml]: changed

File: xml/networking-pcipt_sriov_config.xml (hunk 2 above)
Comment (csymons-suse) [reply xml/networking-pcipt_sriov_config.xml]: clarified
